src/Hocon.API.Tests/HoconAPISpec.ApproveCore.approved.txt

```diff
@@ -1,122 +1,381 @@
 ﻿[assembly: System.Runtime.Versioning.TargetFrameworkAttribute(".NETFramework,Version=v4.6.1", FrameworkDisplayName=".NET Framework 4.6.1")]
-namespace Hocon.Extensions
+namespace Hocon
 {
-    public class static HoconElementExtensions
+    public class static ArrayGetterExtensions
```

Comment:
Too generic of a class name - needs to be made specific to HOCON.

Reply:
OK, I'll append `Hocon` in front of the names.

Reply:
Sounds good